### PR TITLE
Fireperf: Add Fragment Screen Performance Support

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**
## Task
b/228031168  
Merging the EAP branch for fragment performance metrics to master

## Changes aside from the entire EAP branch
1. Set the default fragment sampling rate to 0%
2. Manifest override the dev/e2e test apps sampling rate to 100%
3. Set version code to `20.0.7` (instead of the `20.0.7-eap` that was for the EAP branch)

### CODE REVIEW

**Comment by jeremyjiang-dev** [comment]
LGTM! I am the author so that I cannot LGTM this PR.
@leotianlizhan Feel free to merge the PR once you get all the LGTM

**Comment by visumickey** [comment]
> DO_NOT_SUBMIT

Can we please edit the description of the PR with the following?

1. What are we doing with this PR? (Merging the EAP branch for fragment performance metrics to master)
2. Mention the changes that are going in on top of the changes in the EAP branch
3. "Remove" the DO_NOT_SUBMIT tag since that means that this PR is still in draft state.

File: firebase-perf/src/main/java/com/google/firebase/perf/config/ConfigurationConstants.java [hunk 1]
```diff
@@ -623,4 +623,42 @@ protected String getDeviceCacheFlag() {
...
+    protected static synchronized FragmentSamplingRate getInstance() {
+      if (instance == null) {
+        instance = new FragmentSamplingRate();
+      }
+      return instance;
+    }
+
+    @Override
+    protected Float getDefault() {
+      // Sampling rate range is [0.00f, 1.00f]. By default, sampling rate is 1.00f, which is 100%.
```
Comment (visumickey) [inline firebase-perf/src/main/java/com/google/firebase/perf/config/ConfigurationConstants.java:643]: Can we update the comments here? Currently, the default is set to 0%.

**Review by visumickey — COMMENTED** [review]
In addition to this, can we also make the change to the E2E test apps and the Dev test app to have manifest overrides to set the fragment sampling rate to 100% to make sure we have those apps send the fragment events for processing?

**Review by visumickey — APPROVED** [review]

**Pull request merged**